src/compiler/scala/tools/nsc/typechecker/ContextErrors.scala

```diff
@@ -826,7 +826,7 @@ trait ContextErrors {
       }
 
 
-      case object MacroExpansionException extends Exception with scala.util.control.ControlThrowable
+      case object MacroExpansionException extends ControlThrowable
```

Comment:
One downside to this is that you can't have a `ControlThrowable` be an `Exception`. I don't know if that was needed for logic here or not